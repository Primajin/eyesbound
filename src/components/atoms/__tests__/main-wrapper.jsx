import React from 'react';
import {create} from 'react-test-renderer';

import MainWrapper from '../main-wrapper.jsx';

describe('MainWrapper', () => {
	it('renders correctly', () => {
		const component = create(<MainWrapper/>).toJSON();
		expect(component).toMatchSnapshot();
	});

	it('renders correctly with props', () => {
		const component = create(<MainWrapper><h1>Hello there!</h1></MainWrapper>).toJSON();
		expect(component).toMatchSnapshot();
	});
});
