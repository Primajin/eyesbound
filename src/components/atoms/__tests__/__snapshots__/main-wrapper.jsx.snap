// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainWrapper renders correctly with props 1`] = `
<main
  css={
    Object {
      "map": undefined,
      "name": "14akp75",
      "next": undefined,
      "styles": "
	width: clamp(16rem, 90vw, 70rem);
	margin: 170px auto 100px;
	padding: 0 1.5rem;

	h1, h3 {
		font-weight: 400;
		padding-bottom: 1em;
	}
",
      "toString": [Function],
    }
  }
>
  <h1>
    Hello there!
  </h1>
</main>
`;

exports[`MainWrapper renders correctly without props 1`] = `
<main
  css={
    Object {
      "map": undefined,
      "name": "14akp75",
      "next": undefined,
      "styles": "
	width: clamp(16rem, 90vw, 70rem);
	margin: 170px auto 100px;
	padding: 0 1.5rem;

	h1, h3 {
		font-weight: 400;
		padding-bottom: 1em;
	}
",
      "toString": [Function],
    }
  }
/>
`;
